apps/rush-lib/assets/rush-init/common/config/rush/.npmrc

```diff
@@ -0,0 +1,12 @@
+# Rush uses this file to configure the package registry, regardless of whether the
+# package manager is PNPM, NPM, or Yarn.  Prior to invoking the package manager,
+# Rush will always copy this file to the folder where installation is performed.
+# When NPM is the package manager, Rush works around NPM's mishandling of

```

Comment:
>mishandling [](start = 59, length = 11)

Do we want to explicitly say "mishandling?"

Reply:
I changed it to "NPM's processing of"

---
In reply to: [217268804](https://github.com/Microsoft/web-build-tools/pull/823#discussion_r217268804) [](ancestors = 217268804)